doc/coadd2d.rst

```diff
@@ -19,6 +19,32 @@ part of the data reduction process, although it can
 combine (without weighting) multiple exposures
 during reductions (see :ref:`2d_combine`).
 
+.. note::
+
+        Because the flux of the single reduced science frames is expressed in ``counts``,
+        coadding frames with different exposure times is not recommended.  If the user still
+        wishes to do so, the fluxes of the individual frames are rescaled by the median
+        exposure time. For example, if we have four frames (``a``, ``b``, ``c``, ``d``) with
+        exposure times of ``1800``, ``1800``, ``1800``, and ``1200`` seconds, the exposure
+        time of the coadded frame will be:
+
+        .. code-block:: ini
+
+            coadd_exptime = np.percentile([1800,1800,1800,1200],50, interpolation='higher')
+
+        and the flux of the individual frames will be rescaled by:
+
+        .. code-block:: ini
+
+            rescale_factor = coadd_exptime / exptime
+
+        where ``exptime`` is the exposure time of the individual frames. ``coadd_exptime`` is saved
+        in the header of the coadded frame as ``HIERARCH ALLSPEC2D_EXPTIME``, so that the user can
```

Comment:
I would give the header key without the `HIERARCH` prepended.  I.e., my guess is that to access the value you would do `fits.getheader(file)['ALLSPEC2D_EXPTIME']`, not `fits.getheader(file)['HIARARCH ALLSPEC2D_EXPTIME']`, but I could be wrong.

Reply:
You are right! Thanks, I removed `HIERARCH`